Source/Core/Core/AchievementManager.h

```diff
@@ -155,6 +157,9 @@ class AchievementManager
 private:
   AchievementManager() = default;
 
+  const Common::HttpRequest::Headers USER_AGENT_HEADER = {
+      {"User-Agent", std::string("Dolphin/") + SCM_DESC_STR}};
```

Comment:
This is not used outside the code file and does not change depending on the instance, so you can just put it as a const static file-scope variable in the code file.

Also I haven't tested this but I suspect `std::string("Dolphin/" SCM_DESC_STR)` would also work and actually generate the string at compiletime.